data/json/items/armor.json

```diff
@@ -6411,22 +6411,22 @@
         "weight" : 980,
         "color" : "dark_gray",
         "covers" : ["HEAD"],
-        "to_hit" : 1,
+        "to_hit" : -3,
         "storage" : 0,
         "symbol" : "[",
         "description" : "A spiked helmet once worn by German military officers.  The spike is very sharp.",
         "price" : 24000,
         "material" : ["leather", "iron"],
         "volume" : 7,
-        "cutting" : 0,
+        "cutting" : 4,
         "warmth" : 15,
         "phase" : "solid",
         "environmental_protection" : 0,
         "encumbrance" : 20,
-        "bashing" : 10,
+        "bashing" : 2,
         "coverage" : 70,
         "material_thickness" : 3,
-        "flags" : ["FANCY"]
+        "flags" : ["FANCY", "SPEAR"]
```

Comment:
Please replace ````SPEAR```` with ````STAB````. ````SPEAR```` includes specialized combat messages and special modified stick-in rolls that wouldn't be appropriate for this.